droid-results/src/main/java/uk/gov/nationalarchives/droid/profile/datawriter/ColumnBasedDataWriter.java

```diff
@@ -73,6 +75,28 @@ public void writeDataRowsForOneRowPerFile(List<? extends ProfileResourceNode> no
         csvWriter.flush();
     }
 
+    @Override
+    public void writeJsonForOneRowPerFile(List<? extends ProfileResourceNode> nodes, String[] headers, Writer writer) {
```

Comment:
I think the class `CsvWriterConstants` can be renamed to `WriterConstants` or `OutputWriterConstants`

Reply:
`WriterConstants` is good, I've changed it.